src/rollbar.php

```diff
@@ -5,6 +5,11 @@
  * Unless you need multiple RollbarNotifier instances in the same project, use this.
  */
 class Rollbar {
+    const LEVEL_ERROR = 'error';
+    const LEVEL_CRITICAL = 'critical';
+    const LEVEL_WARNING = 'warning';
+    const LEVEL_INFO = 'info';
+
```

Comment:
My first thought about this was "It ought to be an enum", but PHP doesn't have those.

Could you instead use an abstract class with those const values instead? (per: http://stackoverflow.com/a/254543/456188).

Something like:

```php
abstract class Level {
  const LEVEL_ERROR = 'error';
  const LEVEL_CRITICAL = 'critical';
  const LEVEL_WARNING = 'warning';
  const LEVEL_INFO = 'info';
}
```